keyvault/spring-cloud-azure-starter-keyvault-secrets/single-property-source/terraform/setup_env.sh

```diff
@@ -1 +1 @@
-export ENDPOINT=$(terraform -chdir=./terraform output -raw keyvault_uri)
\ No newline at end of file
+export ENDPOINT=$(terraform -chdir=./terraform output -raw KEYVAULT_URI)
```

Comment:
nit: Add empty line.

Reply:
I suggest ,for setup_env.sh, not add empty line for now.
it will cause issues when running on wsl, yonghui is working on it, and we can do this on yonghui's PR

Reply:
Could you please paste link about related issue or PR?

Reply:
[[Bug] Samples don't work as expected in WSL Azure/azure-sdk-for-java#26301](https://github.com/Azure/azure-sdk-for-java/issues/26301)